src/Menu/BackendMenuBuilder.php

```diff
@@ -37,21 +34,24 @@ class MenuBuilder
     /** @var ContentExtension */
     private $contentExtension;
 
-    public function __construct(FactoryInterface $menuFactory, Config $config, Stopwatch $stopwatch, ContentRepository $contentRepository, UrlGeneratorInterface $urlGenerator, TranslatorInterface $translator, ContentExtension $contentExtension)
-    {
+    public function __construct(
+        FactoryInterface $menuFactory,
+        Config $config,
+        ContentRepository $contentRepository,
+        UrlGeneratorInterface $urlGenerator,
+        TranslatorInterface $translator,
+        ContentExtension $contentExtension
+    ) {
         $this->menuFactory = $menuFactory;
         $this->config = $config;
-        $this->stopwatch = $stopwatch;
         $this->contentRepository = $contentRepository;
         $this->urlGenerator = $urlGenerator;
         $this->translator = $translator;
         $this->contentExtension = $contentExtension;
     }
 
-    public function createSidebarMenu()
+    private function createSidebarMenu(): ItemInterface
```

Comment:
I would not use sidebar word here. It is just a admin panel menu, it can be in form of hamburger for mobiles, etc.

Reply:
Will do.